src/Storage/Listener/LocalListener.php

```diff
@@ -29,7 +27,7 @@ class LocalListener extends AbstractListener {
  * @var array
  */
 	protected $_defaultConfig = [
-		'pathBuilder' => 'BasePath',
+		'pathBuilder' => 'Local',
```

Comment:
It's the convention. LocalListener for Local adapter should use LocalPathBuilder :).
I'd make BasePathBuilder abstract anyway...